backend/services/context_service.py

```
@@ -0,0 +1,335 @@
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)